lib/jsdom/living/nodes/HTMLInputElement-impl.js

```diff
@@ -827,7 +829,12 @@ class HTMLInputElementImpl extends HTMLElementImpl {
           if (typeof number !== "number" || isNaN(number)) {
             return false;
           }
-          return (number - this._stepBase) % allowedValueStep !== 0;
+
+          const isIntegralMultiple = 
+            new Decimal(this._stepBase - number)
```

Comment:
the spec says `and that number subtracted from the step base`, so this follows the wording of the spec more closely although I find it bizzare since it would always result in a negative number - it doesn't matter for a `modulo` operation, it's odd nevertheless.

Reply:
Agreed. I'd honestly just use `number - this._stepBase` which makes more sense, and leave a comment – if that's okay with you.

Reply:
that should be fine, I'll make the change shortly, thank you.